core/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.mapzen.core">
+    package="com.mapzen">
```

Comment:
Either one is fine with me. I was just curious if you ran into any issues the old package.